m4/ax_cxx_compile_stdcxx.m4

```diff
@@ -960,3 +973,34 @@ namespace cxx17
 #endif  // __cplusplus < 201703L
 
 ]])
+
+
+dnl  Tests for new features in C++20
+
+m4_define([_AX_CXX_COMPILE_STDCXX_testbody_new_in_20], [[
+
+#ifndef __cplusplus
+
+#error "This is not a C++ compiler"
+
+#elif __cplusplus < 202002L
+
+#error "This is not a C++20 compiler"
+
+#else
+
+#include <version>
+
+namespace cxx20
+{
+
+// As C++20 supports feature test macros in the standard, there is no
+// immediate need to actually test for feature availability on the
+// Autoconf side.
```

Comment:
I think you'll still need to test for the feature test macros to be available in general? For example via:


```cpp
    namespace language_feature_test {
        static_assert(__cpp_conditional_explicit > 0L);
    }

Reply:
I guess one could do that, however not implemented feature macros are equivalent to all particular feature macro not being announced. So for user code little would change if it checks the feature macros. I guess for code that does not use feature macros in C++17 mode, it might simplify checks.